openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
You will need to reuse the InitTenant function for the tenant stuff to work at all. 

Meaning it you probably want to use the same logcallback function in there. But the function use a more extended logging then what you currently support in that function.